app/main.py

```diff
@@ -1,8 +1,31 @@
 class Person:
-    # write your code here
-    pass
-
+    people = {}
```

Comment:
The `people` dictionary is used to store references to `Person` instances by name. This approach assumes that names are unique, which might not always be the case. Consider using a more robust identifier if possible.